t/plugin/proxy-cache/disk.t

```diff
@@ -706,3 +706,115 @@ GET /t
 --- error_code: 400
 --- response_body eval
 qr/failed to check the configuration of plugin proxy-cache err/
+
+
+
+=== TEST 28: check purge method (route no purge method)
```

Comment:
It is not a good idea to add new test cases to the `proxy-cache` plugin.

We should use a new test file.